irr/src/CIrrDeviceSDL.cpp

```diff
@@ -705,6 +705,7 @@ bool CIrrDeviceSDL::run()
 
 	while (!Close && wrap_PollEvent(&SDL_event)) {
 		// os::Printer::log("event: ", core::stringc((int)SDL_event.type).c_str(),   ELL_INFORMATION);	// just for debugging
+		memset(&irrevent, 0, sizeof(irrevent));
```

Comment:
`std::memset` requires the `<cstring>` header.

Reply:
`irrString` includes `string.h`, thus LGTM.

Reply:
I don't think it is a good idea to rely on transitive includes.